src/openforms/authentication/tests/test_signals.py

```diff
@@ -75,6 +83,77 @@ def test_attribute_set_for_demo_plugin_with_staff_user(self):
         instance.refresh_from_db()
         self.assertEqual(instance.auth_info.value, "123")
 
+    def test_attributes_set_with_registrator_subject_info(self):
+        register = Registry()
+        register("organization_plugin")(ProvidesEmployeePlugin)
+        instance = SubmissionFactory.create()
+        request = factory.get("/foo")
+        request.user = StaffUserFactory.build()
+        request.session = {
+            FORM_AUTH_SESSION_KEY: {
+                "plugin": "organization_plugin",
+                "attribute": AuthAttribute.employee_id,
+                "value": "my-employee-id",
+            },
+            REGISTRATOR_SUBJECT_SESSION_KEY: {
+                "attribute": AuthAttribute.bsn,
+                "value": "123",
+            },
+        }
+
+        with mock_register(register):
+            set_auth_attribute_on_session(
+                sender=None, instance=instance, request=request
+            )
+
+        instance.refresh_from_db()
+
+        # check the property doesn't alias both
+        self.assertNotEqual(instance.auth_info, instance.registrator)
+
+        # check the clients info is stored as .registrator
+        self.assertEqual(instance.auth_info.attribute, AuthAttribute.bsn)
+        self.assertEqual(instance.auth_info.value, "123")
+        # TODO check what we want from this
+        # self.assertEqual(instance.auth_info.plugin, "registrator")
```

Comment:
let's leave it unspecified - as long as existing tests don't break, we don't provide any guarantees about it.